src/programs/display_error.py

```diff
@@ -1,9 +1,10 @@
+from typing import List
 import numpy as np
 import src.data.state as state
 from src.algs import draw_text
```

Comment:
imports should be alphabetical
```suggestion
import numpy as np
import src.data.state as state

from src.algs import draw_text
from typing import List
```